### PR TITLE
Feature/panel/mytickets

### PR DESCRIPTION
**Pull request by ChuntaoFu**
I added some look up data myself. So you probably need to insert some look up data in your end if you want to see your tickets.

### CODE REVIEW

**Comment by TylerPflueger** [comment]
You need to merge develop into this branch to ignore the bower_components you comitted.

File: app/assets/stylesheets/common/main.scss [hunk 1]
```diff
@@ -41,7 +64,7 @@
         &.menu {
 
             &.vertical {
-                width: 300px !important;
+                width: 400px !important;
```
Comment (tpflueger) [inline app/assets/stylesheets/common/main.scss]: If you changed the height, won't this move the tabbed buttons out of the view?

File: db/migrate/20150306035309_add_userid_to_tickets.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUseridToTickets < ActiveRecord::Migration
+  def change
+    add_column :tickets, :user_id, :integer
```
Comment (tpflueger) [inline db/migrate/20150306035309_add_userid_to_tickets.rb]: This file should be removed, the db already associates user_id to a ticket.

File: db/migrate/20150226031628_create_tickets.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@ def change
       t.string :description
       t.string :user
       t.boolean :is_active
+      t.integer :user_id
```
Comment (tpflueger) [inline db/migrate/20150226031628_create_tickets.rb:8]: Tickets already have an association with user_id.

File: app/assets/javascripts/panel/search/_panel.search.html [hunk 4]
```diff
@@ -1 +1,7 @@
-<h1>Search</h1>
\ No newline at end of file
+<div class="ui search">
```
Comment (tpflueger) [inline app/assets/javascripts/panel/search/_panel.search.html:1]: This isn't needed since RJ task is the ticket search.

**Pull request closed without merging**